src/main/java/mate/jdbc/lib/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,136 @@
+package mate.jdbc.lib;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Properties;
+import mate.jdbc.DataProcessingException;
+import mate.jdbc.model.Manufacturer;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new DataProcessingException("can't create a jdbc driver ", e);
+        }
+    }
+
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertFormatRequest = "INSERT INTO Manufacturers(name, country)"
+                + " VALUES(?,?)";
+        int nameIndex = 1;
+        int countryIndex = 2;
+        int idIndex = 1;
```

Comment:
```suggestion
        int nameIndex = 1;
        int countryIndex = 2;
        int idIndex = 1;
```

lets move this to the class constant